doc/nrf/installation/install_ncs.rst

```diff
@@ -248,15 +256,31 @@ For more information about the repository and development model, see the :ref:`d
 
       #. Determine the identifier of the revision of the |NCS| you want to work with.
          See the table above for more information.
-         |install_latest_version|
+         The identifier will be referred to as *nRFConnectSDK_revision* in the next steps.
+
+         .. note::
+              |install_latest_version|
+
+      #. On the command line, open the directory :file:`ncs`.
+         By default, this is one level up from the location where you installed the toolchain.
+         This directory will hold all |NCS| repositories.
+
+      #. Create a directory for the *nRFConnectSDK_revision* you want to use and enter it:
```

Comment:
Why? it is so much simple to instruct the user to run:

`west init -m https://github.com/nrfconnect/sdk-nrf --mr *nRFConnectSDK_revision* *nRFConnectSDK_revision*`

So for example:

`west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.8.0 v2.8.0`

Reply:
Also, this change is required in the other 2 sub-sections below as well